pkg/receive/config.go

```diff
@@ -290,7 +290,7 @@ func parseConfig(content []byte) ([]HashringConfig, error) {
 
 // hashAsMetricValue generates metric value from hash of data.
 func hashAsMetricValue(data []byte) float64 {
-	sum := md5.Sum(data)
+	sum := sha256.Sum256(data)
```

Comment:
I am not sure if this change is necessary. Sha256 is more secure than md5. However, it is not used for the security purpose here. So md5 is fine and Sha256 is even slower.

Reply:
@yeya24 is right. Is there any way you can ignore Muse to getting us this error if we know about this?

Reply:
@bwplotka I don't think so. Other option could be to use SHA1 or SHA2 algorithm. But if we are going to see the speed, than MD5 is the fastest.

Reply:
Yes of course. Also xxhash is faster than md5. Thanks @roidelapluie
Should I go with xxhash? @yeya24 @bwplotka

Reply:
These tools are supposed to help us. I don't understand why we should just change things to make some tool happy.

Reply:
@kakkoyun xxhash is just a suggestion if we want faster algorithm.

Reply:
I was questioning the overall purpose of the PR. I'm ok to try xxhash.

Reply:
Yaa that I agree with you. :) If we don't need it and if the tool is detecting, we can ignore that.

Reply:
So should I move ahead trying xxhash or close this PR?